vizro-core/changelog.d/20231005_145819_maximilian_schulz_86_absence_of_openpyxl_library_causes_console_errors.md

```diff
@@ -0,0 +1,41 @@
+<!--
+A new scriv changelog fragment.
+
+Uncomment the section that is right (remove the HTML comment wrapper).
+-->
+
+<!--
+### Removed
+
+- A bullet item for the Removed category.
+
+-->
+
+### Added
+
+- Raise `ModuleNotFoundError` in case the `export_data` action is used, the export type is `xlsx`, but neither `openpyxl` or `xlsxwriter` are installed. ([#97](https://github.com/mckinsey/vizro/pull/97))
```

Comment:
```suggestion
- Raise `ModuleNotFoundError` in case the `export_data` action is used with `file_format="xlsx"` and neither `openpyxl` nor `xlsxwriter` are installed ([#97](https://github.com/mckinsey/vizro/pull/97))
```
(Do we have a consistent use of full stops on these changelog snippets? Generally seems like we don't include them? Very pedantic I know but would be tidy...)

Reply:
We haven't made any guideline on this yet... It can be altered in the release PR, when all the fragments are collected together. For me that is acceptable for now.

The only guidelines we follow so far:
- focus on what would be important primarily to a user
- use active language (add, change, guide, raise...)